lib/api/2.0/tags.js

```diff
@@ -10,18 +10,6 @@ var _ = injector.get('_'); // jshint ignore:line
 var Errors = injector.get('Errors');
 var parseUrl = require('url').parse;
 
-function getTagName(req) {
-    //TODO: sway 1.0.0 implements a fix for this, but it is lacking in the ^0.6.0 versions
-    //      so we duplicate the 1.0.0 implementation here.
-    // Ref: https://github.com/apigee-127/sway/commit/1b7acd9743aae2f2f94057d3f8911e2af72614f5
-    var pathObject = req.swagger.operation.pathObject;
-    var pathMatch = pathObject.regexp.exec(parseUrl(req.url).pathname);
-    var value = decodeURIComponent(pathMatch[_.findIndex(pathObject.regexp.keys, function(key) {
```

Comment:
Why we drop the tag URL decode phase here? To upgrade swagger-express-mw from 0.6 to 0.7 is too risky which may involve other compatibility issue. Did we test the APIs which also depends on this lib?
Please refer to the comments:
https://github.com/RackHD/on-http/pull/272/files

Reply:
@nortonluo The comments in https://github.com/RackHD/on-http/pull/272/files support our upgrading to 0.7. So I don't understand why you mentioned the comments in https://github.com/RackHD/on-http/pull/272/files.